scripts/cnode-helper-scripts/env

```diff
@@ -392,13 +392,15 @@ checkUpdate() {
       printf "${FG_RED}ERROR${NC}: '${1}' update failed! Please use prereqs.sh to force an update or manually download the file from GitHub"
       return 2
     fi
-    TEMPL_CMD=$(awk '/^# Do NOT modify/,0' "${PARENT}/${1}")
-    TEMPL2_CMD=$(awk '/^# Do NOT modify/,0' "${PARENT}/${1}".tmp)
-    if [[ "$(echo ${TEMPL_CMD} | sha256sum)" != "$(echo ${TEMPL2_CMD} | sha256sum)" ]]; then
-      if [[ $2 = Y ]] || { [[ -t 1 ]] && getAnswer "\nThe static content from ${FG_YELLOW}${1}${NC} does not match with guild-operators repository, do you want to download the updated file?"; }; then
+    OLD_STATIC=$(awk '/#!/{x=1}/^# Do NOT modify/{exit} x' "${PARENT}/${1}")
+    OLD_TEMPL=$(awk '/^# Do NOT modify/,0' "${PARENT}/${1}")
+    GIT_STATIC=$(awk '/#!/{x=1}/^# Do NOT modify/{exit} x' "${PARENT}/${1}".tmp)
+    GIT_TEMPL=$(awk '/^# Do NOT modify/,0' "${PARENT}/${1}".tmp)
+    NEW_STATIC=$(checkUserVariables "${OLD_STATIC}" "${GIT_STATIC}")
+    if [[ "$(echo ${OLD_STATIC} | sha256sum)" != "$(echo ${NEW_STATIC} | sha256sum)" || "$(echo ${OLD_TEMPL} | sha256sum)" != "$(echo ${GIT_TEMPL} | sha256sum)" ]]; then
```

Comment:
Thinking from UX, this could cause twice prompts:
1. To see that there's an update (when this env gets merged to alpha). User pressing yes will quit their gLV
2. The next attempt will again show an update (this time coz NEW_STATIC is different)
Having said that, I cant think of a way to avoid that from happening and not have quiet update, so I believe the one time double prompt could be managed via comms

Reply:
1. I have thought about this before as well when there is an update to both env and gLV. First, it updates env, then quits, then updates gLV and quits. Not the best UX. After the update of the script, before quitting, could we re-launch the script with `exec "<path to self>"`? To be done in the calling script as env just return with exit code 1. 
2. True, for the first time switching from old env to this it will give notification twice about the update. But as you said, not sure I see a way around this either.

Reply:
Adding exec could be a bit tricky as it will return control back to an outdated script that will be handled differently on different OS

Can just be a reminder for ourselves to communicate it when merged to master branch